packages/story-editor/src/storyEditor.js

```diff
@@ -58,6 +59,9 @@ function StoryEditor({ config, initialEdits, children }) {
   const _config = useMemo(() => deepMerge(defaultConfig, config), [config]);
   const { storyId, isRTL, flags } = _config;
 
+  // Tell modals where to set aria-hidden when modals are open
+  setAppElement('body > div:first-of-type');
```

Comment:
Why was this added? 🤔 

We call `setAppElement` already within `wp-story-editor` and within each individual test. Not here.

Reply:
yup sounds good!